src/commcare_cloud/manage_commcare_cloud/list_vault_keys.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import print_function
+from itertools import chain
+
+from commcare_cloud.commands.command_base import CommandBase
+from commcare_cloud.environment.main import get_environment
+from commcare_cloud.environment.paths import get_available_envs
+
+
+class ListVaultKeys(CommandBase):
+    command = 'list-vault-keys'
```

Comment:
Should this get added to the readme?

Reply:
Interesting. I don't have auto-generated documentation for `manage-commcare-cloud` yet, and I'm okay for the time being just because it's a newer, less-used, more in flux entry point that I'm still figuring out the scope fore.